homeassistant/components/emoncms/sensor.py

```diff
@@ -198,22 +206,22 @@ def __init__(
             self._state = None
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the sensor."""
         return self._name
 
     @property
-    def native_unit_of_measurement(self):
+    def native_unit_of_measurement(self) -> str | None:
         """Return the unit of measurement of this entity, if any."""
         return self._unit_of_measurement
 
     @property
-    def native_value(self):
+    def native_value(self) -> str | None:
         """Return the state of the device."""
         return self._state
 
     @property
-    def extra_state_attributes(self):
+    def extra_state_attributes(self) -> dict[str, Any]:
```

Comment:
Can you use shorthand attributes instead: `self._attr_extra_state_attributes`?

Reply:
Done